tasks/update.yml

```diff
@@ -22,4 +22,10 @@
     list_files: yes
   when: "{{ lookup('file', '/tmp/dokuwiki/VERSION')  != lookup('file', '{{ dokuwiki_install_path }}/VERSION') }}"
 
-- debug: msg="{{ file_list }}"
\ No newline at end of file
+- name: Install xml-php package used by dokuwiki.
```

Comment:
This is already done in the install task file. I know it is not always being included. Maybe there is a better way to split the tasks to avoid redundancy.